blinky-statik/src/main/java/org/spideruci/analysis/statik/flow/data/SourceICFGNode.java

```diff
@@ -0,0 +1,54 @@
+package org.spideruci.analysis.statik.flow.data;
+
+public class SourceICFGNode {
+
+	private int lineNum;
+	private String className;
+	private String methodName;
+	
+	public SourceICFGNode(int lineNum, String className, String methodName){
+		this.lineNum = lineNum;
+		this.className = className;
+		this.methodName = methodName;
+	}
+	
+	public void setLineNum(int lineNum){
+		this.lineNum = lineNum;
+	}
+	
+	public void setClassName(String name){
+		className = name;
+	}
+	
+	public void setMethodName(String name){
+		methodName = name;
+	}
+	
+	public int getLineNum(){
+		return lineNum;
+	}
+
+	public String getClassName(){
+		return className;
+	}
+	
+	public String getMethodName(){
+		return methodName;
+	}
+	
+
+	@Override
+	public String toString(){
+		return "LineNum: " + lineNum + " className: " + className + " methodName: " + methodName;
+	}
+	
+	@Override
+	public boolean equals(Object o){
```

Comment:
@jycamus90 - you want to implement the `hashCode` as well, otherwise this object cannot be used in HashMaps and HashSets.